.tekton/ova-provider-server-dev-preview-pull-request.yaml

```
@@ -0,0 +1,616 @@
+        workspace: git-auth
+    - name: prefetch-dependencies
+      params:
+      - name: input
+        value: $(params.prefetch-input)
+      - name: SOURCE_ARTIFACT
+        value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
+      - name: ociStorage
+        value: $(params.output-image).prefetch
+      - name: ociArtifactExpiresAfter
+        value: $(params.image-expires-after)
+      - name: dev-package-managers 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)